document/src/vespa/document/update/tensormodifyupdate.cpp

```diff
@@ -0,0 +1,136 @@
+// Copyright 2019 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```

Comment:
Please use new copyright header:
// Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.